project_tiles.php

```diff
@@ -8,14 +8,25 @@
 
 <div class="container">
 
-	<div class="row" style="padding:2% 1% 1% 1%;">
-		<div class="col s12 m4 l4; valign-wrapper" style="vertical-align: middle; height:50px;">
-			<h5 class="left brown-text text-lighten-2 text-shadow: 2px 2px 7px #111111">
-				Choose a project
-			</h5>
-		</div>
-		<script>var statusFilter = 0, typeFilter = 0;</script>
-		<div class="col s12 m4 l4 valign-wrapper" style="vertical-align: middle; height:50px; padding:1% 1% 1% 1%;">		
+	<div class="row" style="padding:3% 1% 1% 1%;">
+	<script>var statusFilter = 0, typeFilter = 0;</script>
+		<div class="col s12 m4 l4 center-align valign-wrapper" style="vertical-align: middle; height:50px; padding:1% 1% 1% 1%;">		
+	
+    		    <!-- Dropdown Trigger -->
+      		<a class="dropdown-trigger btn donor-background" style="display: block; margin: 0 auto;" href='#' data-target='dropdown3' id='statusFilter'>Filter by Funder</a>
```

Comment:
Change Funder to Partner?

Reply:
probably a good idea but I didn't want to confuse funding parters and village partners.

Reply:
changed it